modules/network-observability-operator-install.adoc

```diff
@@ -34,17 +34,13 @@ This documentation assumes that your `LokiStack` instance name is `loki`. Using
 . Navigate to the *Flow Collector* tab, and click *Create FlowCollector*. Make the following selections in the form view:
 .. *spec.agent.ebpf.Sampling*: Specify a sampling size for flows. Lower sampling sizes will have higher impact on resource utilization. For more information, see the "FlowCollector API reference", `spec.agent.ebpf`.
 .. If you are using Loki, set the following specifications:
-... *spec.loki.enable*: Select the check box to enable storing flows in Loki.
-... *spec.loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network`. The first part of the URL, "loki", must match the name of your `LokiStack`.
-... *spec.loki.authToken*: Select the `FORWARD` value.
-... *spec.loki.statusUrl*: Set this to `https://loki-query-frontend-http.netobserv.svc:3100/`. The first part of the URL, "loki", must match the name of your `LokiStack`.
+... *spec.loki.mode*: Set this to `LokiStack` mode.
+... *spec.loki.lokistack.name*: Set this to `loki`.
 ... *spec.loki.tls.enable*: Select the checkbox to enable TLS.
 ... *spec.loki.statusTls*: The `enable` value is false by default.
```

Comment:
Those two should also be removed

Reply:
Ah ok, I was following QE CRD https://github.com/openshift/openshift-docs/pull/67836#discussion_r1401262883, but I think some things are there that don't align quite with what we want to recommend in the doc. I will remove this. I was initially thinking this needed to be removed but then I saw it was still in the QE CRD